### PR TITLE
WebAssembly | Control flow - update `return` and `select`

### PR DESCRIPTION
**Pull request by ghost**
Cooperative PR to https://github.com/mdn/content/pull/18838; these may be merged out of sync/order.

Updates the `return` control flow instruction example to show discarding values from the Wasm value stack.

Adds an example for `select t`, using WebAssembly external reference types.

### CODE REVIEW

File: live-examples/wat-examples/statements/return.wat [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
   )
-
+  (func (export "get_second_value") (param i32 i32) (result i32)
+    ;; load both arguments onto the stack
+    local.get 0
+    local.get 1
+    ;; return the second argument; the first is discarded
+    return
+  )
 )
```
Comment (MendyBerger) [inline live-examples/wat-examples/statements/return.wat:10]: I don't think two examples are need here, this having one example with 2 values on the stack is enough.

**Review by MendyBerger — CHANGES_REQUESTED** [review]
Should not be merged before https://github.com/mdn/bob/pull/842

**Comment by ghost** [comment]
@schalkneethling May we have have a review here, as you merged the blocker PR?

File: live-examples/wat-examples/statements/select.wat [hunk 2]
```diff
@@ -1,15 +1,18 @@
 (module
-  (import "console" "log" (func $log (param i32)))
-  (func $main
+  (func (export "select_10_or_20") (result i32)
```
Comment (MendyBerger) [inline live-examples/wat-examples/statements/select.wat]: Maybe rename the functions to something like `select_simple` and `select_externref`?

**Review by MendyBerger — APPROVED** [review]

**Comment by ghost** [comment]
@Josh-Cena Seeing as you assigned the reviewer, may we get someone to review for merging?

**Review by Josh-Cena — APPROVED** [review]
Oh—let's go then

**Pull request merged**